build-bench-env.sh

```diff
@@ -765,6 +785,7 @@ if test "$setup_redis" = "1"; then
     echo "$devdir/redis-$version_redis already exists; no need to download it"
   else
     wget --no-verbose "http://download.redis.io/releases/redis-$version_redis.tar.gz"
```

Comment:
I think this link can be upgraded to https :)

Reply:
Indeed! :)